identity-admin/webapp/components/AccountActions/DeleteAccount.tsx

```diff
@@ -1,16 +1,21 @@
-import React, { useState } from 'react';
+import React, { MutableRefObject, useState } from 'react';
 import ReactModal from 'react-modal';
 import { DangerButton, Button } from '../Button';
 import { useDeleteAccount } from '../../hooks/useDeleteAccount';
 import styles from './DeleteAccount.module.css';
 import { useRouter } from 'next/router';
-import { User } from '../../interfaces';
 
 ReactModal.setAppElement('#__next');
 
-type DeleteAccountProps = Pick<User, 'userId'>;
+type DeleteAccountProps = {
+  userId: number;
+  modalRef: MutableRefObject<HTMLDivElement | null>;
+};
```

Comment:
This type looks fine, but it's causing a TS error in the test file:

```
components/AccountActions/DeleteAccount.test.tsx:11:39 - error TS2741: Property 'modalRef' is missing in type '{ userId: number; }' but required in type 'DeleteAccountProps'.

11 const renderComponent = () => render(<DeleteAccount userId={123} />);
                                         ~~~~~~~~~~~~~

  components/AccountActions/DeleteAccount.tsx:12:3
    12   modalRef: MutableRefObject<HTMLDivElement | null>;
         ~~~~~~~~
    'modalRef' is declared here.
```

Mocking the ref in the tests seems to work:

```ts
// components/AccountActions/DeleteAccount.test.tsx
// Line 11

const renderComponent = () => {
  const mockRef = React.createRef<HTMLDivElement>();
  return render(<DeleteAccount userId={123} modalRef={mockRef} />);
};
```